ios/Classes/PMNotificationManager.m

```diff
@@ -54,50 +54,69 @@ - (void)photoLibraryDidChange:(PHChange *)changeInstance {
     return;
   }
 
-  PHFetchResultChangeDetails *details = [changeInstance changeDetailsForFetchResult:result];
+  PHFetchResultChangeDetails *details =
+      [changeInstance changeDetailsForFetchResult:result];
 
-  NSDictionary *detailResult = [self convertChangeDetailsToNotifyDetail:details];
+  NSMutableDictionary *detailResult =
+      [self convertChangeDetailsToNotifyDetail:details];
 
-  [PMLogUtils.sharedInstance info: [NSString stringWithFormat:@"on change result = %@", detailResult]];
+  NSUInteger oldCount = result.count;
   [self refreshFetchResult];
+  NSUInteger newCount = result.count;
+  detailResult[@"oldCount"] = @(oldCount);
+  detailResult[@"newCount"] = @(newCount);
 
```

Comment:
Other parts of the changes are caused by the program automatic format, effective modification only here